docs/faq.md

```diff
@@ -56,28 +56,31 @@ APIs that already exist as Kubernetes resources, such as:
 * ad hoc scripting ("make an X for every Y")
 * configuration abstraction ("when I say A, that means {X,Y,Z}")
 * higher-level automation of custom APIs added by [Operators][operator]
+* gluing an [external CRUD API][] into the Kubernetes control plane with a
+  simple translation layer
 
 [controllers]: /concepts/#controller
 [catset]: /examples/#catset-javascript
 [operator]: https://coreos.com/operators/
+[external CRUD API]: #can-i-call-external-apis-from-my-hook
 
 ### What is Metacontroller not good for?
 
-Metacontroller is currently not a good fit for "bridging" other APIs
-(like a cloud provider or app-specific API) into the Kubernetes API,
-for example by representing it as a CRD and doing 2-way reconciliation.
-If something changes in the external system, Metacontroller won't know
-that it needs to call your hook again unless you set it to [poll][resync period].
-
-That's because the initial controller patterns we support are focused on use
-cases where both the inputs and outputs of your controller can be expressed as
-Kubernetes API objects (either built-in or custom).
-If you have a concrete use case that involves reconciling external state,
-we'd appreciate if you [file an issue][issues] describing it so we can
-work on defining additional patterns.
-
-[resync period]: /api/compositecontroller/#resync-period
-[issues]: {{ site.repo_url }}/issues
+Metacontroller is not a good fit when you need to examine a large number of
+objects to answer a single hook request.
+For example, if you need to be sent a list of all Pods or all Nodes in order to
+decide on your desired state, we'd have to call your hook with the full list of
+all Pods or Nodes any time any one of them changed.
+However, it might be a good fit if your desired behavior can be naturally
+broken down into per-Pod or per-Node tasks, since then we only need to send your
+hook the objects that have changed.
```

Comment:
... your hook _to_ the objects that have changed ...?

Reply:
It's the other way around. We send the objects to the hook; we don't send the hook to the objects.

I can see how the wording is ambiguous though. Do you think it would be easier to read if it were reworded like this?

> ... we only need to send the objects that have changed to your hook.

Reply:
Oh, misunderstood the description. How about `... we only need to send the changed objects to your hook.`?